modules/ROOT/pages/sftp/sftp-documentation.adoc

```diff
@@ -106,189 +110,233 @@ An FileSystemProvider which provides instances of SftpFileSystem from instances
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | {nbsp}
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | {nbsp}
 |===
 
 
-==== For Configurations
+==== For Configurations.
 * <<config>> {nbsp}
 
 ==== Throws
 * SFTP:RETRY_EXHAUSTED {nbsp}
 * SFTP:ILLEGAL_PATH {nbsp}
-* SFTP:CONNECTIVITY {nbsp}
 * SFTP:FILE_ALREADY_EXISTS {nbsp}
+* SFTP:CONNECTIVITY {nbsp}
 
 
 [[createDirectory]]
 === Create Directory
 `<sftp:create-directory>`
 
-+++
-Creates a new directory on directoryPath.
-+++
+
+Creates a new directory on `directoryPath`.
+
 
 ==== Parameters
 [cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | *x*{nbsp}
-| Directory Path a| String |  +++The new directory's name.+++ |  | *x*{nbsp}
+| Directory Path a| String |  The new directory's name. |  | *x*{nbsp}
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | {nbsp}
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | {nbsp}
 |===
 
 
-==== For Configurations
+==== For Configurations.
 * <<config>> {nbsp}
 
 ==== Throws
+
 * SFTP:RETRY_EXHAUSTED {nbsp}
-* SFTP:ACCESS_DENIED {nbsp}
 * SFTP:ILLEGAL_PATH {nbsp}
-* SFTP:CONNECTIVITY {nbsp}
+* SFTP:ACCESS_DENIED {nbsp}
 * SFTP:FILE_ALREADY_EXISTS {nbsp}
+* SFTP:CONNECTIVITY {nbsp}
 
 
 [[delete]]
 === Delete
 `<sftp:delete>`
 
-+++
-Deletes the file pointed by path, provided that it's not locked
-+++
+Deletes the file that the path points to, provided that it's not locked.
+
 
 ==== Parameters
 [cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | *x*{nbsp}
-| Path a| String |  +++The path to the file to be deleted.+++ |  | *x*{nbsp}
+| Path a| String |  The path to the file to delete. |  | *x*{nbsp}
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | {nbsp}
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | {nbsp}
 |===
 
 
 ==== For Configurations
+
 * <<config>> {nbsp}
 
 ==== Throws
+
 * SFTP:RETRY_EXHAUSTED {nbsp}
-* SFTP:ACCESS_DENIED {nbsp}
 * SFTP:ILLEGAL_PATH {nbsp}
+* SFTP:ACCESS_DENIED {nbsp}
 * SFTP:CONNECTIVITY {nbsp}
 
 
 [[list]]
 === List
 `<sftp:list>`
 
-+++
-Lists all the files in the directoryPath that match the given matcher. If the listing encounters a directory, the output list includes its contents depending on the value of the recursive parameter. If recursive is set to true but a found directory is rejected by the matcher, then there isn't any recursion into that directory.
-+++
+
+Lists all the files in the `directoryPath` that match the given matcher. 
```

Comment:
Should be:
Lists all the files in the **Directory Path** that depending on the rules in the **File Matching Rules** field.